.eslintrc.base.js

```diff
@@ -5,22 +5,53 @@ module.exports = {
         sourceType: 'module',
         project: 'tsconfig.json',
     },
-    plugins: ['@typescript-eslint', 'extra-rules'],
-    extends: ['plugin:@typescript-eslint/recommended', 'plugin:prettier/recommended', 'prettier/@typescript-eslint'],
+    plugins: ['@typescript-eslint', 'import'],
+    extends: [
+        'eslint:recommended',
+        'plugin:@typescript-eslint/eslint-recommended',
+        'plugin:@typescript-eslint/recommended',
+        'plugin:import/errors',
+        'plugin:import/warnings',
+        'plugin:import/typescript',
+        'prettier/@typescript-eslint',
+    ],
     rules: {
         'max-classes-per-file': 'off',
         '@typescript-eslint/no-non-null-assertion': 'off',
         '@typescript-eslint/interface-name-prefix': 'off',
         '@typescript-eslint/no-empty-interface': 'off',
-        '@typescript-eslint/no-explicit-any': 'off',
```

Comment:
has this rule been changed in one of the configurations we extend? or has it been covered by a different rule now?